EU4ToVic2Tests/MapperTests/IdeaEffectsMapperTests.cpp

```diff
@@ -32,8 +32,8 @@ TEST(Mappers_IdeaEffectMapperTests, getArmyInvestmentFromIdeaReturnsZeroForMissi
 	std::stringstream input;
 	mappers::IdeaEffectMapper theMapper(input);
 
-	auto investment = theMapper.getArmyInvestmentFromIdea("missingIdea", 1);
-	ASSERT_EQ(investment, 0);
+	auto investment = theMapper.getArmyFromIdea("missingIdea", 1);
+	ASSERT_EQ(investment, 5);
```

Comment:
Should rename test to match (same for other modified tests)

Reply:
all tests rewritten an renamed.